app/js/services/attachment-service.js

```diff
@@ -0,0 +1,14 @@
+const AttachmentService = ($resource, $http) => {
+
+    let obj = $resource('https://dev-uprep.nijel.org/api/observation_evidences/:id', {
```

Comment:
We need to create a base URL rather than duplicate it across all the services. This will also make it easier for testing locally as we would not have to go and replace all the urls in all the services